apps/etterlatte-behandling/src/main/kotlin/behandling/aktivitetsplikt/AktivitetspliktRoute.kt

```diff
@@ -288,7 +288,7 @@ internal fun Route.aktivitetspliktRoutes(
         get("/ny") {
             logger.info("Henter ny aktivitetsplikt vurdering for oppgaveId=$oppgaveId")
             val vurdering =
-                inTransaction { aktivitetspliktService.hentVurderingForOppgave(oppgaveId) }
+                inTransaction { aktivitetspliktService.hentVurderingForOppgaveGammel(oppgaveId) }
```

Comment:
Er ikke dette uheldig med tanke på at det er `ny`-endepunktet? Vil ikke det brekke ting for 12-måneders-vurdering?